apps/condo/domains/user/components/UserMenu/MobileUserMenu.tsx

```diff
@@ -28,27 +26,27 @@ export const StyledMenu = styled(Menu)`
   transform: translate(-5%, 10px);
 `
 
-const modalStyle: ComponentProps<typeof Modal>['style'] = {
-    position: 'absolute',
-    left: 0,
-    right: 0,
-    bottom: 0,
-    pointerEvents: 'initial',
-    top: 'initial',
-    transformOrigin: 'initial',
-    maxWidth: '100%',
-}
+const StyledModal = styled(Modal)`
```

Comment:
I'm not sure, that it should be done with styled modal...

Btw, aren't we switching from emotion to css/less.modules?